conf/conf.go

```diff
@@ -97,6 +97,15 @@ func GetAssetId(ctx context.Context, config apiserver.Configuration, projId stri
 	return common.Ptr(int32(dbAssets[0].ID)), nil
 }
 
+func InsertAsset(ctx context.Context, config apiserver.Configuration, projId string, SerialNumber string, assetId int32) error {
+	var dbAsset dbkentix.Sensor
```

Comment:
Maybe InsertSensor is the better function name. InsertAsset implies to save an asset in Eliona.

Reply:
Then dbAsset should be dbSensor